docs/devenv-local.md

```diff
@@ -22,6 +22,7 @@ extra software needs installing and configuring
 ```bash
 sudo dnf install etcd docker git libvirt-devel
 sudo dnf install golang glide golang-googlecode-tools-goimports
+sudo dnf install kubernetes-client
```

Comment:
We shouldn't need to install this, because we build the 'kubectl' binary from git - a few lines later where we add

 export PATH=\$GOPATH/bin:\$GOPATH/src/k8s.io/kubernetes/_output/bin:\$PATH

The \$GOPATH/src/k8s.io/kubernetes/_output/bin  directory should contain kubectl.

Reply:
Yes it's true, I probably forgot to source my .bashrc.